__snapshots__/manifest.js

```diff
@@ -0,0 +1,604 @@
+exports['Manifest pullRequest boostraps from HEAD manifest if first PR: changes'] = `
```

Comment:
don't forget to peek in here to make sure these snapshots look like what you'd expect from the manifest/aggregate releaser